stores/sql/rows.go

```diff
@@ -12,7 +12,7 @@ type Scanner interface {
 
 type ContractRow struct {
 	FCID    FileContractID
-	HostID  int64
+	HostID  *int64
```

Comment:
Will a contract ever not be attached to a host where nil would make sense?

Reply:
Yes, we merged `contracts` and `archived_contracts` into a single `contracts` table. If a host we used before goes offline for an extended period of time we remove it (`RemoveOfflineHosts`), so it's possible for a row in the contracts table to point to a host that no longer exists.